src/test/java/se/sundsvall/casedata/api/HistoryResourceTest.java

```diff
@@ -50,21 +69,34 @@ void getAttachmentHistory() {
 			.getResponseBody();
 
 		// Assert
-		assertThat(response).isEqualTo(history);
+		JSONAssert.assertEquals(
+			"[{\"commitMetadata\":null,\"changeType\":\"PROPERTY_VALUE_CHANGED\",\"propertyName\":\"property\",\"right\":\"oldValue\",\"propertyAdded\":false,\"propertyRemoved\":false,\"propertyValueChanged\":true,\"propertyNameWithPath\":\"property\",\"affectedLocalId\":\"id\",\"affectedObject\":null,\"affectedGlobalId\":{\"typeName\":\"entity\",\"cdoId\":\"id\"}}]",
```

Comment:
`Changes.class` has no no-arguments constructor so  `.expectBody(Changes.class)` throws deserialization errors. I guess we could create a api-model and map the JaVers provided data into to that, probably

Reply:
Yes, I think we should do this regardless, since it's a bad idea to depend on a third-party class (that we do not control) in our API.

There is a risk that our API will change after a dependebot-lift (of the javers dependency) if we keep the class as the response-model.